src/twisted/application/runner/test/test_pidfile.py

```
@@ -474,3 +474,107 @@ def remove(self):
 
     def exists(self):
         return self._exists
+
+
+    def sep(self):
+        """
+        @see: L{IFilePath.sep}
+
+        Not implemented
```

shouldn't this raise NotImplemented error?
In the case in which some new test is using it, it will raise an explicit error, rather than a generic error about the None type missing some string attributes.

---

Also I think it's better to have no docstring rather than one containing only `@see`, since if there is no docstring, pydoctor will inherit the docstring from the superclass/interface.

---

@mthuurne I put the @see docstring in there, because when looking directly at the code, it is sometimes difficult to figure out what method comes from what interface.  I can take out the docstring though if people don't want it.

---

I guess for test code it doesn't matter much, but for modules that end users are going to be reading the documentation of, I think it's better to have the inherited documentation inlined by pydoctor, since that means it is accessible without having to navigate to a different page when reading the docs as HTML.